targets/zephyr/README.md

```diff
@@ -36,3 +36,50 @@ The following boards are supported by the Zephyr target configurations:
 
 - ESP32 DevKitC WROOM <esp32_devkitc_wroom>
 - ESP32 S3 <esp32s3_devkitc/esp32s3/procpu>
+
+## Testing using Emulators (QEMU and NATIVESIM)
```

Comment:
@JonesKisaka How about those other dependencies that were in the `emulator_test.md`?